tests/unit/uuid.ts

```diff
@@ -7,9 +7,8 @@ const root = global as any;
 
 describe('uuid util', () => {
```

Comment:
Since it's not a uuid util anymore, can we rename all uuid references? Also, it seems character length is important for consistency. Can we add a test that checks for the length?